app/src/main/java/org/mozilla/fenix/wallpapers/Wallpaper.kt

```diff
@@ -49,14 +49,19 @@ sealed class Wallpaper {
         abstract val expirationDate: Date?
         abstract val remoteParentDirName: String
         @Suppress("MagicNumber")
+        /**
+         * A promotional partnered wallpaper.
+         */
         data class House(
             override val name: String,
             override val expirationDate: Date? = Calendar.getInstance().run {
                 set(2022, Calendar.APRIL, 30)
                 time
             }
-        ) : Remote() {
+        ) : Remote(), Promotional {
             override val remoteParentDirName: String = "house"
+            override fun isAvailableInLocale(locale: String): Boolean =
+                listOf("en-US", "es-US").contains(locale)
```

Comment:
In case this just happens to be useful information, we can tell a user's region based on their `store.state.search.region`. If we were trying to make the wallpaper available for all of US instead of just based on their locale. See https://github.com/mozilla-mobile/fenix/pull/17580/files#diff-3101d2ecb99584cfe0d93898bc7eee5a629ddf6de9f2d2632f56fb3f04e2c34dR395 for an example.

Reply:
That's nice!!!
We could add it as a follow up ticket :)

Reply:
That is actually really good to know. I think when we talked about this initially, we didn't think there was another good way for us to determine locale. That's why we included es-US, since it's another high usage locale in the US.